index.html

```diff
@@ -4,6 +4,10 @@
     <title>MetaMask &lt; = &gt; Ledger Bridge</title>
   </head>
   <body>
-    <script src="./bundle.js" type="text/javascript"></script>
+    <script type="text/javascript">
+      const script = document.createElement('script');
```

Comment:
@darkwing Can you explain the need for this change a bit further?

Reply:
This has been for cache-busting during the debugging process -- it will be rolled back when we confirm everything is working properly.